content/docs/concepts/security/index.md

```diff
@@ -815,6 +814,56 @@ spec:
     name: "products-viewer"
 {{< /text >}}
 
+### Using Istio authorization on plain TCP protocols
+
+The examples in [Service role](#servicerole) and [Service role binding](#servicerolebinding) show the
+typical way to use Istio authorization on services using the HTTP protocol. In those examples, all fields
+in a service role and service role binding are supported.
+
+Istio authorization supports services using any plain TCP protocols, such as MongoDB. In this case,
+you configure the service roles and service role bindings in the same way you did for the HTTP service.
+The difference is that certain fields, constraints and properties are only applicable to HTTP services.
+These fields include:
+
+    * The `paths` and `methods` fields in the service role configuration object.
```

Comment:
Do not indent the list. If you review the rendered content \[here.\]\(https://deploy-preview-2612--preliminary-istio.netlify.com/docs/concepts/security/) As is, the list is rendered as a literal code block.